enclave_manager/avalon_enclave_manager/singleton/singleton_enclave_manager.py

```diff
@@ -257,6 +270,26 @@ def _execute_work_order(self, input_json_str):
 
         return json_response
 
+# -------------------------------------------------------------------------
+
+    def _handle_wo_process_failure(self, msg, wo_id):
+        """
+        Handle failure of work order by creating proper response JSON
+        and updating database appropriately.
+
+        Parameters:
+            @param msg - Error message to put into response
+            @param wo_id - Id of the work-order being handled
+        """
+        wo_response = dict()
+        wo_response["error"] = dict()
+        wo_response["error"]["code"] = WorkOrderStatus.FAILED
+        wo_response["error"]["message"] = msg
```

Comment:
Can we use the function  `create_error_response() ` in 
 https://github.com/hyperledger/avalon/blob/master/common/python/utility/jrpc_utility.py#L20